src/prefect/storage/bitbucket.py

```diff
@@ -18,7 +18,8 @@
     total=10,
     backoff_factor=0.5,
     status_forcelist=[429],
-    method_whitelist=["HEAD", "GET", "OPTIONS"],
+    # ignoring new allowed_methods param until mypy update
```

Comment:
Just a minor change here. We're quite unlikely to upgrade mypy for this repository.
```suggestion
    # typeshed is out of date with urllib3 and missing `allowed_methods`
```

Reply:
Done!